### ISSUE TITLE
Rename auth-server extension

### ISSUE THREAD

**Issue by tkrille** [osiam/osiam#47]
As discussed in https://github.com/osiam/osiam/pull/44#issuecomment-150343601, we should rename the extension `urn:org.osiam:scim:extensions:auth-server`. It should be changed to `urn:org.osiam:scim:extensions:osiam` or `urn:org.osiam:scim:extensions:auth` or something? Seems safe to automatically migrate with Flyway.

**Comment by dacrome** [comment]
Should we [register an official scim urn](https://tools.ietf.org/html/rfc7643#section-10.3)? Independent from this decision, I think we should integrate into the scim namespace: `urn:ietf:params:scim:schemas:osiam`

**Comment by tkrille** [comment]
Yes, why not.

**Comment by tkrille** [comment]
Blocked by #165

**Comment by dacrome** [comment]
So I like to rename it to `urn:ietf:params:scim:schemas:osiam`.

**Comment by tkrille** [comment]
I'm more in favor for `urn:ietf:params:scim:schemas:org.osiam`. I think extensions have to be namespaced by your domain to get registered. Of course, that's something the people from SCIM have to decide, but I think it's better. Did you already start the registration process?

**Comment by dacrome** [comment]
Yes, you are right. `urn:ietf:params:scim:schemas:org.osiam` should be registered.

> Did you already start the registration process?

No, but I check the process next week.

**Comment by tkrille** [comment]
@dacrome Do you have new information on this?

**Comment by dacrome** [comment]
@tkrille: No. I just ( ;) ) need to trigger the [registration process](https://tools.ietf.org/html/rfc7643#section-10).

**Comment by tkrille** [comment]
I thought about this and I somehow don't understand, why we need an extensions for this information. Wouldn't it be better to internally store it in some new field in the database? I mean, nobody should need this information outside of OSIAM and certainly nobody should be allowed to change it. So we could do the following: Just leave the extension at it is and migrate it to a new field in the next release. This would also reduce the scope of 3.0 and make an earlier release possible. What do you think @dacrome and @wallner?

**Comment by tkrille** [comment]
@dacrome @wallner Should we do it like proposed by me and just close this issue?

**Comment by tkrille** [comment]
Alright, this is closed now and here's the new issue for the migration: https://github.com/osiam/osiam/issues/242
